app/serializers/api/v1/user_serializer.rb

```diff
@@ -7,6 +7,7 @@ class UserSerializer < ApplicationSerializer
     has_one :image, serializer: ImageSerializer
     has_one :address, serializer: AddressSerializer
     has_many :user_roles, serializer: UserRoleSerializer
+    # has_many :organisations_users, serializer: OrganisationsUserSerializer
```

Comment:
I assume this was forgotten : )

Reply:
Nope, this was breaking the `current_user_profile` (api/v1/auth/current_user_profile)  response with error
`PG::DuplicateAlias: ERROR: WITH query name "organisations_user_ids_by_users" specified more than once`